Documentation/network/concepts/routing.rst

```diff
@@ -31,6 +31,9 @@ Requirements on the network
   Cilium nodes can already reach each other, all routing requirements are
   already met.
 
+* The underlying network must support IPv4. See `issue 17240 <https://github.com/cilium/cilium/issues/17240>`_
```

Comment:
Nit: We've got a dedicated rST role to format GH issue references consistently, can you please use it?:

```suggestion
* The underlying network must support IPv4. See :gh-issue:`17240`
```